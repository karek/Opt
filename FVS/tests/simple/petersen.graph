a b
b c
c d
d e
e a
a p
b q
c r
d s
e t
p r
r t
t q
q s
s p
