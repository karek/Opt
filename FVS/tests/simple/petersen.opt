3
a c s 
