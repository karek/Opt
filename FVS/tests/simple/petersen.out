3
e r s 
